CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,162 @@
+* An imposed vacation 
+* A permanent or temporary ban from some or all Quansight spaces (mailing lists, GitHub repos, in-person events, etc.)
+* Assistance to the complainant with a report to other bodies, for example, institutional offices or appropriate law enforcement agencies
+* Removing a person from Quansight employment or other formal affiliation
+* Publishing an account of the harassment and calling for the resignation of the alleged harasser from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar)
+* Any other response that Quansight deems necessary and appropriate to the situation
+* At Quansight events, if a participant engages in behavior that violates this Code of Conduct, the organizers and staff may take any action they deem appropriate
+
+Potential consequences for violating the Quansight Code of Conduct at an in-person company event include:
+* Warning the person to cease their behavior and that any further reports will result in sanctions
+* Requiring that the person avoid any interaction with, and physical proximity to, the person they are harassing for the remainder of the event
+* Immediately ending any event volunteer responsibilities and privileges the reported person holds
+* Expelling the person from the event
+* Requiring that the person immediately leave the event and not return
+* Banning the person from future events (either indefinitely or for a certain time period)
+* Any other response that Quansight deems necessary and appropriate to the situation
+
+No one espousing views or values contrary to the standards of our Code of Conduct will be permitted to hold any position representing Quansight, including volunteer positions. Quansight has the right and responsibility to remove, edit, or reject comments, commits, code, edits, and other online contributions that are not aligned with this Code of Conduct.
+
+We aim to respond within one week to the original reporter with either a resolution or an explanation of why the situation is not yet resolved. We will contact the person who is the subject of the report to let them know what actions will be taken as a result of the report if any.
+
+Our policy is to make sure that everyone aware of the initial incident is also made aware that official action has been taken while still respecting the privacy of individuals. Quansight may choose to make a public report of the incident while maintaining the anonymity of those involved.
+
+
+## 3. License
+
+This code of conduct has been adapted from numerous sources, including TODO Group Code of Conduct template, Django Code of Conduct, Contributor Covenant, Ubuntu Code of Conduct v2.0, NumFOCUS Code of Conduct, Vox Media Code of Conduct, the Bokeh Code of Conduct, the SciPy Code of Conduct, the Carpentries Code of Conduct, and the NeurIPS Code of Conduct.
```

Comment:
There's still a TODO in here that looks like it needs deleting

Reply:
I thought that too, but then I came across this https://todogroup.org/blog/open-code-of-conduct/

Reply:
🤦 what a name for an entity

Reply:
Sorry for the confusion, but yes, "TODO Group" is a thing and I referred to the template they provided for CoCs.

Reply:
FYI, I've added hyperlinks to all the documents referenced so people can find them easily.